packages/app/src/systems/Account/components/AccountList/AccountList.tsx

```diff
@@ -32,6 +32,13 @@ export function AccountList({
     setShowHidden((s) => !s);
   }
 
+  const collator = new Intl.Collator(undefined, {
+    numeric: true,
+    sensitivity: "base"
+  });
+
+  accounts = accounts.sort((a, b) => collator.compare(a.name, b.name));
+
   return (
```

Comment:
also, could you add a short and clear changelog for this fix?

just run `pnpm changeset` and follow the instruction (it looks like a patch fix)